src/main/java/com/devcycle/sdk/server/openfeature/DevCycleProvider.java

```diff
@@ -0,0 +1,83 @@
+package com.devcycle.sdk.server.openfeature;
+
+import com.devcycle.sdk.server.common.api.IDevCycleClient;
+import com.devcycle.sdk.server.common.model.DevCycleUser;
+import com.devcycle.sdk.server.common.model.Variable;
+import dev.openfeature.sdk.*;
+import dev.openfeature.sdk.exceptions.ProviderNotReadyError;
+
+public class DevCycleProvider implements FeatureProvider {
+    private static final String PROVIDER_NAME = "DevCycleProvider";
```

Comment:
@jonathannorris - what is this field supposed to be? Should it identify the SDK type of cloud/local?

Reply:
Most of the other providers just provided a simple name:

Launch Darkly: 
https://github.com/launchdarkly/openfeature-java-server/blob/351302ca0e0d4e48646dc51e74ef956dd984ffde/src/main/java/com/launchdarkly/openfeature/serverprovider/Provider.java#L30

CloudBees:

https://github.com/rollout/cloudbees-openfeature-provider-java/blob/e371a40cbfa9a57e05aa53d69f36e6d8a92f7e19/src/main/java/com/cloudbees/openfeature/provider/CloudbeesProvider.java#L26